src/lib/pages/frontsite/PatronProfile/PatronShow.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Container, Grid, Icon } from 'semantic-ui-react';
+
+export const ShowAllLink = ({ items, onShowAll, onShowLess, rowsPerPage }) => {
+  const allHits = items.total === items.hits.length ? true : false;
+  return (
+    items.total > 0 &&
+    items.total > rowsPerPage && (
+      <Grid>
+        <Grid.Column textAlign="center">
+          {!allHits ? (
+            <Container
+              className="link-like top-spaced"
+              onClick={() => onShowAll()}
+            >
+              SHOW ALL <Icon name="caret right" />
+            </Container>
+          ) : (
+            <Container className="link-like" onClick={() => onShowLess()}>
+              SHOW LESS <Icon name="caret right" />
+            </Container>
+          )}
+        </Grid.Column>
+      </Grid>
+    )
+  );
+};
+
+ShowAllLink.propTypes = {
+  items: PropTypes.object.isRequired,
+  onShowAll: PropTypes.func.isRequired,
```

Comment:
Very minor:
```suggestion
  onShowAllClick: PropTypes.func.isRequired,
```
same below